test/legacy-upgrade/check-from-v0.111.0-kafka-sink.td

```diff
@@ -7,17 +7,14 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-$ set-from-sql var=expected-compression-implicit-create-sql
-SELECT sql FROM expected_compression_implicit_create_sql
-
 > SHOW CREATE SINK compression_implicit;
-"materialize.public.compression_implicit" ${expected-compression-implicit-create-sql}
+materialize.public.compression_implicit "CREATE SINK materialize.public.compression_implicit IN CLUSTER quickstart FROM materialize.public.kafka_sink_from INTO KAFKA CONNECTION materialize.public.kafka_conn (TOPIC = 'kafka-sink') FORMAT JSON ENVELOPE DEBEZIUM;"
```

Comment:
Dumb question, but why do some of the SHOW CREATE outputs have newlines and some don't?

Reply:
`sql-pretty` doesn't support every statement kind. E.g., `to_doc` [has a case](https://github.com/MaterializeInc/materialize/blob/bf9abe7fd897f8c45324d951e0caf16ca395f462/src/sql-pretty/src/lib.rs#L38) for `CreateSource`, but not for `CreateSink`.